features/reporting.feature

```diff
@@ -0,0 +1,10 @@
+Feature: Covering letter
+  As Pension Wise
+  We want to store customer details for reporting
+  So we can use the result to provide a better service
+
+  Scenario: Details of the appointment are stored for reporting
+    Given a customer has had a Pension Wise appointment
+    When we send them their summary document
```

Comment:
The flick between `we` and `I` here and elsewhere reads a little off. Can we settle on referring to `we` or `I` consistently?